pom.xml

```diff
@@ -23,10 +24,10 @@
     </licenses>	
 
     <properties>
-        <jenkins.version>1.651.3</jenkins.version>
-        <java.level>7</java.level>
+        <jenkins.version>2.361.4</jenkins.version>
+        <java.level>17</java.level>
```

Comment:
The improve a plugin tutorial advises that this should be removed, since it is no longer relevant.
```suggestion
```